### DISCUSSION TITLE
I want to contribute

### DISCUSSION THREAD

**Discussion by joelbqz** [strapi/strapi#7122] [Legacy Topics]
Hey guys, I love Strapi and I would like to write a guide on how to deploy it to Dokku(please, let me know if this is something already covered by someone else). I'm just still not used to Github, I still don't understand how pull requests to documentation, etc works. If you can point me on the right direction, I will be happy to write this guide.

**Comment by sandiprb** [comment]
Hey @joelbqz  I think you can get started by sharing your Strapi knowledge with community on multiple channels like Answering Stackoverflow questions, participating in Strapi discussions and various other. Just do it :)

**Comment by lauriejim** [comment]
Hello! Thank you @joelbqz for your message.
The documentation is in the mono repo under the `docs` folder.
After that you can see the architecture of the folder to understand where to put files / guides.

About Dokku, it's not really an hosting platform but a tool.
So could be under `Optional Software Guides` section.
Ping @derrickmehaffy, you agree w/ that?

**Reply by derrickmehaffy** [reply]
Yes @joelbqz Dokku is basically just a PaaS using docker (as I understand it) functionally similar to a self-hosted version of Heroku.

I think creating a spot within the optional software guides would be the best place, we should clarify to those reading it, the purpose. Currently the only guides there relate to proxy software, let me know if you have issues creating a guide within those docs and I can certainly help you with the layout structure.
